### PR TITLE
Remove alpha max from solver

### PR DESCRIPTION
**Pull request by QB3**
It removes the parameter alpha_max from the solver function, tackling #7

### CODE REVIEW

**Comment by mathurinm** [comment]
The diff contains unrelated stuff, try to avoid this @QB3 as it make reading it more difficult

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,105 @@
...
+*.woff
+*.woff2
+*.bak
+*.dat
+*.eot
+*.examples
+*.buildinfo
+*.md5
+*.dir
+/doc/generated/*.rst
```
Comment (mathurinm) [inline .gitignore:32]: I think it is safe to fully ignore doc/*  @ qb3

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,105 @@
...
+.*.sw[op]
+
+# LaTeX output
+*.dvi
+*.pdf
+*.ps
+*.png
+
+# LaTeX cruft
+*.aux
```
Comment (mathurinm) [inline .gitignore:55]: do we need those ?

File: .gitignore [hunk 3]
```diff
@@ -1,6 +1,105 @@
 clar.egg-info/*
 clar/__pycache__/*
+sg_execution_times.rst
 
+.vscode/
+
+# n umpy
+*.npy
+
+# web pages
```
Comment (mathurinm) [inline .gitignore:10]: I'd rather keep the gitignore short but that's personal

File: .gitignore [hunk 4]
```diff
@@ -1,6 +1,105 @@
 clar.egg-info/*
 clar/__pycache__/*
+sg_execution_times.rst
```
Comment (mathurinm) [inline .gitignore:3]: is this file needed ?

File: README.rst [hunk 5]
```diff
@@ -3,19 +3,65 @@ CLaR
...
 
-It also implements a variation of the MRCE algorithm with a L21 penalization on the regression coefficient Beta, and which take in account the repetitions see
-Rothman, A. J., Levina, E., & Zhu, J. (2010). Sparse multivariate regression with covariance estimation. Journal of Computational and Graphical Statistics, 19(4), 947-962.
 
+Documentation
+=============
 
-To be able to run the code you first need to run, in this folder:
-```pip install -e .```
+Please visit https://qb3.github.io/CLaR/ for the latest version
```
Comment (mathurinm) [inline README.rst:12]: this is unrelated to the PR

File: clar/tests/test_clar.py [hunk 6]
```diff
@@ -31,8 +31,8 @@ def test_clar(
     print("alpha = %.2e" % alpha)
     print("alpha = %.2e" % alpha)
     print("sigma_min = %.2e" % sigma_min)
-    B_sgcl_me, S_inv_sgcl_me, E, gaps_me = solver(
-        X, all_epochs, alpha, alpha_max, sigma_min, B0=None,
+    _, _, _, gaps_me = solver(
```
Comment (mathurinm) [inline clar/tests/test_clar.py:34]: maybe you can use my_call(...)[-1]

File: clar/tests/test_mrce.py [hunk 7]
```diff
@@ -23,7 +23,7 @@ def test_update_sigma_glasso():
 
     alpha_prec = 0.001
 
-    covariance, precision = update_sigma_glasso(
+    _, _ = update_sigma_glasso(
```
Comment (mathurinm) [inline clar/tests/test_mrce.py:26]: no need to return the result

File: clar/tests/test_mrce.py [hunk 8]
```diff
@@ -37,7 +37,7 @@ def test_mrce():
     pb_name = "mrce"
     tol = 1e-4
 
-    X, all_epochs, B_star, (_, S_star) = get_data_me(
+    X, all_epochs, _, (_, _) = get_data_me(
```
Comment (mathurinm) [inline clar/tests/test_mrce.py:40]: heavy syntax IMO

File: clar/tests/test_mtl.py [hunk 9]
```diff
@@ -60,8 +60,8 @@ def test_mtl_me():
     alpha_div = 1.1
     alpha = alpha_max / alpha_div
 
-    B_mtl, S_inv, E, gaps = solver(
-        X, all_epochs, alpha, alpha_max, sigma_min, B0=None,
+    _, _, _, gaps = solver(
```
Comment (mathurinm) [inline clar/tests/test_mtl.py:63]: ditto

**Comment by QB3** [comment]
PR too dirty, I started a new one, see #17
